code/game/objects/items/devices/scanners/sequence_scanner.dm

```diff
@@ -15,65 +15,78 @@
 	throw_speed = 3
 	throw_range = 7
 	custom_materials = list(/datum/material/iron=200)
+
 	var/list/discovered = list() //hit a dna console to update the scanners database
 	var/list/buffer
 	var/ready = TRUE
 	var/cooldown = 200
 
-/obj/item/sequence_scanner/attack(mob/living/M, mob/living/carbon/human/user)
+/obj/item/sequence_scanner/attack(mob/living/mob, mob/living/carbon/human/user)
 	add_fingerprint(user)
-	if (!HAS_TRAIT(M, TRAIT_GENELESS) && !HAS_TRAIT(M, TRAIT_BADDNA)) //no scanning if its a husk or DNA-less Species
-		user.visible_message(span_notice("[user] analyzes [M]'s genetic sequence."), \
-							span_notice("You analyze [M]'s genetic sequence."))
-		gene_scan(M, user)
-
+	//no scanning if its a husk or DNA-less Species
+	if (!HAS_TRAIT(mob, TRAIT_GENELESS) && !HAS_TRAIT(mob, TRAIT_BADDNA))
+		user.visible_message(span_notice("[user] analyzes [mob]'s genetic sequence."), \
+							span_notice("You analyze [mob]'s genetic sequence."))
```

Comment:
```suggestion
		user.visible_message(
				span_notice("[user] analyzes [mob]'s genetic sequence."),
				span_notice("You analyze [mob]'s genetic sequence."),
		)
```
The indentation for this message is really annoying, though You're not actually touching it much fixing it would be nice